src/main/resources/application.properties

```diff
@@ -0,0 +1,3 @@
+api.endpoint=/company/{companyNumber}/objections/strike-off
```

Comment:
i know we've done it this way for the other apps but i found it a bit of a pain when looking back at them recently, to find out what the base endpoint was. So, would it be better to place this endpoint back into the controller classes to make them more readable - the reader then doesn't have to go and look up this constant value.

Reply:
Makes sense, will change. Assume there isn't a valid, long lost reason why we did it this way previously.

Reply:
I think it was my idea originally in extensions, just thought it was a way of having the base_url in one place , avoids typos etc..  Maybe it is still a good idea, downside is the controllers aren't as readable. Maybe see what the others think?

Reply:
Leave it as is, seems a common way of doing it on stack overflow